--- main.wrl	2012-01-13 18:20:02.000000000 +0900
+++ main_fs.wrl	2012-01-20 19:03:53.000000000 +0900
@@ -521,7 +521,8 @@
 						 jointType "rotate"
 						 jointId 8
 						 jointAxis 1.0 0.0 0.0 
-						 translation -0.040 0.0 -0.090
+						 #translation -0.040 0.0 -0.090
+						 translation -0.099 0.0 -0.090
 						 ulimit [ 2.8448866807507569 ] #  163
 						 llimit [-2.8448866807507569 ] # -163
 						 uvlimit [ 5.2359877559829888 ] #  300
@@ -749,7 +750,8 @@
 						 jointAxis 1.0 0.0 0.0 
 						 #  translation -0.0335 0 -0.09 
 						 #translation 0.0 0.0 -0.090 
-						 translation -0.040 0.0 -0.090 
+						 #translation -0.040 0.0 -0.090 
+						 translation -0.099 0.0 -0.090 
 						 ulimit [ 2.8448866807507569 ] #  163
 						 llimit [-2.8448866807507569 ] # -163
 						 uvlimit [ 5.2359877559829888 ] #  300
